docs/howto/how-to-build-runtime.md

```diff
@@ -157,7 +157,7 @@ $ tree -L 3 ./Product/out
 
 ```
 
-Here, let's recall that the main target of our project is the arm architecture. If you have a development environment running Linux for arm on a device made of an arm CPU, such as Odroid-XU4, you will get a runtime binary that can be run on the arm architecture with the same command above. This is the simplest way to get a binary for an arm device. However, in most cases, native builds on arm devices are too impractical as they require too long. Therefore, we will create an executable binary of an architecture other than x86_64 through a `cross build`. For cross-build method for each architecture, please refer to the corresponding document in the following section, [How to cross-build runtime for different architevture](#how-to-cross-build-runtime-for-different-architevture).
+Here, let's recall that the main target of our project is the arm architecture. If you have a development environment running Linux for arm on a device made of an arm CPU, such as Odroid-XU4, you will get a runtime binary that can be run on the arm architecture with the same command above. This is the simplest way to get a binary for an arm device. However, in most cases, native builds on arm devices are too impractical as they require too long. Therefore, we will create an executable binary of an architecture other than x86_64 through a `cross build`. For cross-build method for each architecture, please refer to the corresponding document in the following section, [How to cross-build runtime for different architecture](#how-to-cross-build-runtime-for-different-architecture).
```

Comment:
@dev-hue , I don't mean that make this commit from you. Just get know you the usage of '+-' button on toolbar menu of review comment window. If you click the button '+-', you can create a review comment with diff suggestion for fix, and I can accept it by single button click. Please experience with this review comment for next use. :)

** Please don't accept this diff suggestion, It's made for example. Please reject this suggestion if you read this comment.**

```suggestion
Here, let's recall that the main target of our project is the arm architecture. If you have a development environment running Linux for arm on a device made of an arm CPU, such as Odroid-XU4, you will get a runtime binary that can be run on the arm architecture with the same command above. This is the simplest way to get a binary for an arm device. However, in most cases, native builds on arm devices are too impractical as they require too long. Therefore, we will create an executable binary of an architecture other than x86_64 through a `cross build`. For cross-build method for each architecture, please refer to the corresponding document in the following section, [How to cross-build runtime for different architecture](#how-to-cross-build-runtime-for-different-architevture).
```

Reply:
@lemmaa I understood perfectly! 👍 
But I couldn't find **reject button**.
There are just **Commit suggestion** and **Add suggestion to batch**.
It seems that I'm not the author.
If not, I'm sorry, please let me know how to reject. 😢

Reply:
> .. But I couldn't find reject button. ...

@dev-hue , Yes, it's not there. Just ignore this comment and give 'approval' on PR if you agree. :)

Reply:
I found a good guide to this feature. https://haacked.com/archive/2019/06/03/suggested-changes/

Reply:
@lemmaa Thanks for many things!
I learned a lot today :)